development/build/index.js

```diff
@@ -72,6 +73,9 @@ async function defineAndRunBuildTasks() {
     version,
   } = await parseArgv();
 
+  // build lavamoat runtime file
+  await lavapack.buildRuntime({ scuttleGlobalThis: false });
```

Comment:
this should go somewhere, and not be built in place in the package
in some cases we are running multiple instances of the build system in parallel with different build configurations, and they could conflict

Reply:
since the runtime is no longer static (its built based on a parameterization based on build configuration) we should:
- remove it from the static tasks
- modify `lavapack.buildRuntime` to return the string of the built runtime
- generate the runtime in `development/build/scripts` when building with lavamoat
- write it out to the correct destinations for each browser

Reply:
would it be ok to fix this in a newer lavamoat version after merging this PR?

Reply:
Yes fix

Reply:
since lavapack is already published as is, thats fine. be sure to make this requested fix soon

Reply:
https://github.com/LavaMoat/LavaMoat/issues/451 opened issue to solve this